bedrock/firefox/tests/test_views.py

```diff
@@ -755,7 +755,7 @@ def test_wait_face_scene_2(self, render_mock):
     # wait face video experiment bug 1431795
 
     def test_wait_face_video_scene_1(self, render_mock):
-        req = RequestFactory().get('/firefox/new/?xv=2xfaster')
+        req = RequestFactory().get('/firefox/new/?xv=waitface&v=2xfaster')
```

Comment:
[Tests are currently failing in CI](https://circleci.com/gh/mozilla/bedrock/8908?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link). I think you just need to update the params like you did here, for the `test_wait_face_video_scene_2` test directly below.